pep-0517.txt

```diff
@@ -307,6 +313,10 @@ Of course, it's up to users to make sure that they pass options which
 make sense for the particular build backend and package that they are
 building.
 
+The hooks may be called with positional or keyword arguments, so backends
```

Comment:
It's tricky to add new arguments to a callback-based interface, unless you specify that the functions all have to accept arbitrary, unknown `**kwargs` - but then the caller doesn't know whether passing an argument has any effect or not. I think I'd say we should define new hooks if we need to provide extra arguments - it's easier to check for a new hook than to work out what options are accepted for an existing one.